boards/xtensa/odroid_go/odroid_go_defconfig

```diff
@@ -26,3 +26,6 @@ CONFIG_GEN_IRQ_VECTOR_TABLE=n
 
 CONFIG_I2C=y
 CONFIG_CLOCK_CONTROL=y
+
+# required to enable LCD backlight
+CONFIG_REGULATOR=y
```

Comment:
Do you want the backlight enabled even if the display is not enabled (i.e., in the hello_world sample)? Might be better to condition this on `CONFIG_DISPLAY=y`

Reply:
![backlight_odroid_go](https://user-images.githubusercontent.com/1384437/160009461-b1e9ec17-3270-4ec6-adf4-902e5bc73892.png)

According to the schematic where IO14 is connected to a pull down (if I read it correctly) I would expect that the backlight would be off when `CONFIG_REGULATOR=n` but it doesn't. I need to set IO14 to 0 in order to turn it off.
Any ideas what I am doing wrong?